msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
```

Comment:
Heh... You can help! But first let's have something that works. Please note that current Git for Windows does *not* ship those components, though, so that would add to my maintenance burden. Besides, I have no idea whether e.g. posh-git's author would be okay with us shipping their product (there was no attempt from posh-git's side to contact us so far).